sysutils/skhd/files/patch-Makefile.diff

```diff
@@ -0,0 +1,25 @@
+--- makefile.orig	2020-12-03 18:18:40.000000000 -0800
++++ makefile	2020-12-03 18:18:48.000000000 -0800
+@@ -1,6 +1,6 @@
+ FRAMEWORKS     = -framework Carbon -framework CoreServices
+ BUILD_PATH     = ./bin
+-BUILD_FLAGS    = -std=c99 -Wall -g -O0
++CFLAGS         = -std=c99 -Wall -O3
```

Comment:
just as in the other PR, this should likely become `CFLAGS         += -std=c99 -Wall` where the optimization, arch flags and such are automatically set by MacPorts.